tmk_core/protocol/vusb/main.c

```diff
@@ -114,6 +118,11 @@ int main(void) {
     // clock prescaler
     clock_prescale_set(clock_div_1);
 #endif
+
+#ifdef EEPROM_DRIVER
+    eeprom_driver_init();
+#endif
+
```

Comment:
Another part of EEPROM code which does not seem to belong here.

Reply:
See my reply in lufa.c.